File: FitAppReact/ClientApp/src/models/User/UserTrainingParams.ts
//Program powstał na Wydziale Informatyki Politechniki Białostockiej

import { BodyTargetEnum } from "../enums/BodyTargetEnum";
import { DifficultyEnum } from "../enums/DifficultyEnum";
import { TrainingCategoryEnum } from "../enums/TrainingCategoryEnum";
import { TrainingCondition } from "../Training/TrainingCondition";

export interface UserTrainingParams {
  trainingCategory: TrainingCategoryEnum;
  difficulty: DifficultyEnum;
  trainingConditions: TrainingCondition[];
  bodyTarget: BodyTargetEnum;
}

File: FitAppReact.Common/Classes/UserTrainingParams.cs
using FitAppReact.Common.Enums;
using FitAppReact.Common.ModelsDTO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitAppReact.Common.Classes
{
    public class UserTrainingParams
    {
        public TrainingCategoryEnum trainingCategory { get; set; }
        public DifficultyEnum difficulty { get; set; }
        public IEnumerable<TrainingConditionDTO> trainingConditions { get; set; }
        public BodyTargetEnum bodyTarget { get; set; }
    }
}
